src/main/java/core/basesyntax/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int CODE_POSITION = 0;
+    private static final int FRUIT_POSITION = 1;
+    private static final int QUANTITY_POSITION = 2;
+    private static final String SEPARATOR = ",";
+    private static final int PARTS_NUMBER = 3;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+        boolean firstLine = true;
+        for (String line : lines) {
+            if (firstLine) {
+                firstLine = false;
+                continue;
```

Comment:
Using a boolean flag to skip the first line can be error-prone and is not the most readable solution. Consider using a more expressive method, like removing the first line from the list before entering the loop, or starting the iteration from the second element if the first line is always a header.